index.html

```diff
@@ -40,107 +40,91 @@ <h1 class="position-text">Don't let your business be left behind. <br>
+                        will get amazing designs faster.
+                    </p>
+                    <p>
+                        Currently celebrating our 4th year since opening our doors, we have consistently won many awards
+                        for best
+                        practices from a web development agency and many more nominations for fast and effective
+                        outcomes.
+                    </p>
+                    We deliver the impossible and throw out many smiles.
+                    </p>
+                </div>
             </div>
         </section>
 
         <section id="services">
-            <div class="flip-container">
-                <div class="flip-motion">
-                    <div class="front-card">
+            <h2 class="text-center">Our Services</h2>
+            <div class="services-container">
+                <div class="flip-container">
+                    <div class="flip-motion">
+                        <div class="front-card">
 
-                        <p class="front-card-creativity">Web Development</p>
-                    </div>
-                    <div class="back-card-creativity">
-                        <a href="#">
-                            <h2 class="style-cards">web development services</h2>
-                        </a>
-                        <h3 class="info-title style-cards">HTML, CSS and JavaScript</h3>
-                        <p class="detailed-info">Lorem Ipsum is simply dummy text of the printing and typesetting
-                            industry.
-                            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an
-                            unknown
-                            printer took a galley of type and scrambled it to make a type specimen book. It has survived
-                            not
-                            only five centuries, but also the leap into electronic typesetting, remaining essentially
-                            unchanged.
-                            It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum
-                            passages,
-                            and more recently with desktop publishing software like Aldus PageMaker including versions
-                            of
-                            Lorem Ipsum.</p>
+                            <h3 class="front-card-creativity">Web Development</h3>
+                        </div>
+                        <div class="back-card-creativity">
+                            <!-- h3 that will appear only on mobile  -->
+                            <h3 class="front-card-creativity hide">Web Development</h3>
+                            <p class="detailed-info style-cards">Beautiful websites also need to perform optimally – for
+                                both users and search engines.
```

Comment:
I really liked the descriptions you used for the services the agency provides on the clip cards

very imaginative 🙌